modules/contact-form.php

```diff
@@ -13,7 +13,7 @@
  * Additional Search Queries: contact, form, grunion, feedback, submission
  */
 
-include dirname( __FILE__ ) . '/contact-form/grunion-contact-form.php';
+require_once dirname( __FILE__ ) . '/contact-form/grunion-contact-form.php';
```

Comment:
This was throwing an error at one point in the PR as I was activating default modules more than once.